### PR TITLE
Added swal timer APIs documentation and samples

### LINKED ISSUE #41: Add documentation and sample for new timer APIs
Add documentation and sample code for API introduced by: 

https://github.com/sweetalert2/sweetalert2/pull/1325
https://github.com/sweetalert2/sweetalert2/pull/1322

### PR DESCRIPTION
**Pull request by gverni**
Close #41 

I would like to add an example as well. I'll create a new PR for that. I believe it would be better to merge API documentation first (since the APIs are already released) and then we can work on examples (see also https://github.com/sweetalert2/sweetalert2.github.io/issues/43 related to adding more examples)

### CODE REVIEW

**Comment by gverni** [comment]
I managed to add a new timer functions example. In order for this to work the https://github.com/sweetalert2/sweetalert2.github.io/pull/44 needs to be included first

**Comment by gverni** [comment]
Add documentation for `isTimerRunning`:

https://github.com/sweetalert2/sweetalert2/pull/1330

File: index.html [hunk 1]
```diff
@@ -482,6 +482,60 @@ <h3 id="examples">More examples</h3>
...
+    '&lt;button id="stop-timer"&gt;Please stop the timer!!&lt;/button&gt;&lt;br/&gt;' + 
+    '&lt;button id="resume-timer" disabled&gt;Phew... you can restart now!&lt;/button&gt;&lt;br/&gt;' + 
+    '&lt;button id="toggle-timer"&gt;Toggle&lt;/button&gt;',
+    timer: 10000,
+    backdrop: false,
+    onOpen: () => {
+      Swal.showLoading()
+
+      function toggleButtons () {
+        Swal.getContent().querySelector('#stop-timer').disabled = !Swal.getContent().querySelector('#stop-timer').disabled
```
Comment (limonte) [inline index.html]: `Swal.getContent().querySelector('#stop-timer')` is repeated multiple times, that violates the DRY principle, consider creating a variable:

```js
const content = Swal.getContent()
const $ = content.querySelector.bind(content)

const stop = $('#stop-timer')
const resume = $('#resume-timer')
const toggle = $('#toggle-timer')
const increase = $('#increase-timer')
```

File: index.html [hunk 2]
```diff
@@ -482,6 +482,60 @@ <h3 id="examples">More examples</h3>
...
+        Swal.getContent().querySelector('#resume-timer').disabled = !Swal.getContent().querySelector('#resume-timer').disabled  
+      }
+
+      Swal.getContent().querySelector('#stop-timer').addEventListener('click', () => {
+        console.log(Swal.stopTimer())
```
Comment (limonte) [inline index.html]: `console.log` is probably not needed here and below.

File: index.html (hunk 1 above)
Comment (gverni) [reply index.html]: Done in a90f6d6

File: index.html (hunk 2 above)
Comment (gverni) [reply index.html]: Done in e185f9d

File: index.html [hunk 3]
```diff
@@ -482,6 +482,60 @@ <h3 id="examples">More examples</h3>
...
+<code>  let timerInterval
+  Swal({
+    title: 'Auto close alert!',
+    html: 'I will close in &lt;strong&gt;&lt;/strong&gt; seconds.' + '&lt;br/&gt;&lt;br/&gt;' + 
+    '&lt;button id="increase-timer"&gt;I need 5 more seconds!&lt;/button&gt;&lt;br/&gt;' + 
+    '&lt;button id="stop-timer"&gt;Please stop the timer!!&lt;/button&gt;&lt;br/&gt;' + 
+    '&lt;button id="resume-timer" disabled&gt;Phew... you can restart now!&lt;/button&gt;&lt;br/&gt;' + 
+    '&lt;button id="toggle-timer"&gt;Toggle&lt;/button&gt;',
+    timer: 10000,
+    backdrop: false,
```
[the remark this answers is not in the document]
Comment (gverni) [reply index.html]: Done in e185f9d and a90f6d6

**Comment by gverni** [comment]
Thanks @limonte for the very instructive review! I've done all the changes. I had to be a little creative with the `html` string :-) Hope it's still readable. 

I also added 65b6082 because i think that having all the sample boxes the same size makes for a cleaner looking page. Feel free to reject that commit if you don't like it.

**Pull request merged**